TYPO3.TYPO3CR/Resources/Private/Schema/NodeTypes.schema.yaml

```diff
@@ -75,6 +75,10 @@ additionalProperties:
     'options':
       type: dictionary
 
+    'fusion':
```

Comment:
I was almost about to merge this, but this IMHO cannot happen, it implicitly binds CR to Fusion, which it shouldn't be. SORRY :/

The configuration should go under "options" and not be mentioned in this schema.